### PR TITLE
Add endTime to Web UI

### PR DESCRIPTION
**Pull request by yu-yamada**

### CODE REVIEW

**Comment by cberner** [comment]
I don't think adding this to `index.html` is a good idea. There's already a lot of information there, and this would make it more cluttered. How about adding it to the `query.html` page?

**Comment by yu-yamada** [comment]
Thank you for your comment.

Chenged to `query.html`

**Comment by cberner** [comment]
Looks good, will merge this soon

File: presto-main/src/main/resources/webapp/query.html [hunk 1]
```diff
@@ -174,6 +177,10 @@
     d3.select('#sessionCatalog').text(query.session.catalog);
     d3.select('#sessionSchema').text(query.session.schema);
     d3.select('#elapsedTime').text(query.queryStats.elapsedTime);
+    if (query.state == 'FINISHED' || query.state == 'FAILED') {
```
Comment (electrum) [inline presto-main/src/main/resources/webapp/query.html]: This misses other states like ABORTED. Shouldnt we always render the end time if it exists?

Comment (electrum) [inline presto-main/src/main/resources/webapp/query.html]: Actually, for this struct, those might be the only states, but we should just render the end time if it exists.

**Comment by electrum** [comment]
This commit message shows your email as `yu-yamada <yu@yamadayu-no-MacBook-Pro.local>`. Can you please fix that?

File: presto-main/src/main/resources/webapp/query.html (hunk 1 above)
Comment (yu-yamada) [inline presto-main/src/main/resources/webapp/query.html]: OK, I think so.  How about  `if (query.queryStats.endTime)`

**Comment by yu-yamada** [comment]
The email was mistake...  I will fix it.

**Comment by cberner** [comment]
Merged. Thanks!

**Pull request closed without merging**